UMN-Common.psm1

```diff
@@ -736,3 +736,109 @@ function Write-Log {
 		if ($Console -eq $true) {Write-Host $Message}
 
 } #END Write-Log
+
+function Get-RandomCharacter{
+	<#
+	.SYNOPSIS
+	This function is to randomize a list of characters for password generation.
+
+	.PARAMETER length
+	How many of each character set you want.
+
+	.PARAMETER characters
+	A set list of characters to be selected from.
+
+	.EXAMPLE
+	Get-RandomCharacters -length 2 -characters '1234567890'
+	42
+	#>
+
+	param(
+	[Parameter(Mandatory=$true)]$length,
+	[Parameter(Mandatory=$true)]$characters
+	)
+	Begin{}
+	Process{
+		$random = 1..$length | ForEach-Object {Get-Random -Maximum $characters.length}
+		$private:ofs=""
```

Comment:
looks like this sets the default separator when joining an array of items, I think it'd be better to be more explicit in the places where you're joining rather than defining it here (see later comments)